x-pack/plugins/security_solution/public/timelines/hooks/use_create_timeline.tsx

```diff
@@ -47,9 +47,9 @@ export const useCreateTimeline = ({
   onClick,
 }: UseCreateTimelineParams): ((options?: { timeRange?: TimeRange }) => void) => {
   const dispatch = useDispatch();
-  const defaultDataViewSelector = useMemo(() => sourcererSelectors.defaultDataViewSelector(), []);
-  const { id: dataViewId, patternList: selectedPatterns } =
-    useDeepEqualSelector(defaultDataViewSelector);
+  const { id: dataViewId, patternList: selectedPatterns } = useSelector(
+    sourcererSelectors.defaultDataView
+  ) ?? { id: '', patternList: [] };
```

Comment:
NIT: Is it possible for the `sourcererSelectors.defaultDataView` selector to return `undefined`? I see other usages of this selector where this scenario is not controlled. We can probably remove the `?? { id: '', patternList: [] }` default value.

Reply:
not sure why I had this, you are right, removed.

Reply:
A lot of tests blowing up on this line, changing back, not sure if the issue is within this code or the type.